app/views/posts/_form.html.erb

```diff
@@ -1,13 +0,0 @@
-<%= form_for @post do |f| %>
```

Comment:
Since introducing nested resources, this partial caused me more trouble than it was worth...